packages/deploy-tooling/test/unit/cli/config.test.ts

```diff
@@ -17,14 +17,16 @@ describe('cli/config', () => {
 
     describe('getDeploymentConfig', () => {
         test('valid config path', async () => {
-            expect(await getDeploymentConfig(join(__dirname, '../../test-input/ui5-deploy.yaml'))).toBeDefined();
+            expect(
+                await getDeploymentConfig(join(__dirname, '../../fixtures/simple-app/ui5-deploy.yaml'))
+            ).toBeDefined();
         });
         test('invalid config', async () => {
-            await expect(getDeploymentConfig(join(__dirname, '../../test-input/ui5.yaml'))).rejects.toThrowError();
+            await expect(getDeploymentConfig(join(__dirname, '../../simple-app/ui5.yaml'))).rejects.toThrowError();
```

Comment:
Shouldn't the path be `'../../fixtures/simple-app/ui5.yaml'`?  I assume that the test is successful, because anyway an error is thrown since the `ui5.yaml` can't be found.

Reply:
ooops

Reply:
Fixed with 21b7dfd